claude/skills/jira/scripts/fetch-ticket.py

```
@@ -0,0 +1,412 @@
+    """Fetch ticket JSON from Jira REST API."""
+    jira_base = config["jira_base"]
+    jira_host = get_jira_host(config)
+    url = f"{jira_base}/rest/api/2/issue/{key}?fields={FIELDS}"
+    req = urllib.request.Request(url)
+
+    auth = get_auth(jira_host, config)
+    if auth:
+        req.add_header("Authorization", auth)
+
+    try:
+        with urllib.request.urlopen(req) as resp:
```

Ruff (0.15.5)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)